pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 # Note: Various modules refer to this system as "encoded", not "fourfront".
 name = "encoded"
-version = "2.1.8"
+version = "2.2.7"
```

Comment:
Bumping the minor version but decrementing the patch version by one?  I'm not sure why not just 2.2.0 or if no real incompatibilities 2.1.9?

Reply:
Ah you're right, this was a mistake. I'll fix that.

Reply:
But incompatibilities are only relevant for a major version increase, as far as I'm aware. New features merit a minor version increase, and patch version increases are mainly for bug fixes. I wanted to increase the minor version because the url handling in the search tables was a new feature, IMO.